x-pack/plugins/cloud/server/routes/set_default_space_solution.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { SavedObjectsErrorHelpers } from '@kbn/core/server';
+
+import { parseOnboardingSolution } from '../../common/parse_onboarding_default_solution';
+import { createLicensedRouteHandler } from './error_handler';
```

Comment:
I would expect this handler to be inside a "licensed_route" file and not "error_handler".

Reply:
I will rename that!